cob_hardware_config/cob3-2/config/arm_driver.yaml

```diff
@@ -0,0 +1,16 @@
+# powercube_chain parameters
+can_module: PCAN
+can_device: /dev/pcan2
+can_baudrate: 1000
+modul_ids: [4,5,6,7,8,9,10]
+joint_names: [arm_1_joint, arm_2_joint, arm_3_joint, arm_4_joint, arm_5_joint, arm_6_joint, arm_7_joint]
+max_accelerations: [0.8,0.8,0.8,0.8,0.8,0.8,0.8]
+min_publish_duration: 0.1
+horizon: 0.1 # [s]
+frequency: 68
+force_use_movevel: True
+
+# trajectory controller parameters
+ptp_vel: 0.4 # rad/sec
+ptp_acc: 0.2 # rad/sec^2
+max_error: 0.15 # rad
```

Comment:
the trajectory controller parameters should be deleted here as they are in the controller.yaml